grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ts

```diff
@@ -31,36 +27,53 @@ export class OsqueryDetails extends Plugin {
     map(singleItem => singleItem.payload as OsqueryResult),
   );
 
-  readonly resultsTable$ = this.osqueryResults$.pipe(
-    map(result => result?.table),
-  );
-
-  readonly resultsStderr$ = this.osqueryResults$.pipe(
-    map(result => result?.stderr),
-  );
-
   readonly osqueryProgress$: Observable<OsqueryProgress> = this.flowListEntry$.pipe(
     map(listEntry => listEntry.flow.progress as OsqueryProgress),
     filter(isNonNull),
   );
 
+  readonly resultsTable$ = this.osqueryResults$.pipe(
```

Comment:
Please mark all possible members as private, to make the public interface of the component class apparent to the reader of your code.